packages/@tinacms/graphql/src/primitives/resolver/index.ts

```diff
@@ -632,16 +633,18 @@ export class Resolver {
   }
 }
 
-const resolveDateInput = (
-  value: string
-) => {
+const resolveDateInput = (value: string) => {
   /**
    * Convert string to `new Date()`
    */
   const date = new Date(value)
   if (!isValid(date)) {
     throw 'Invalid Date'
   }
+
+  /**
+   * toISOString() converts to UTC
+   */
   return date.toISOString()
```

Comment:
I don't believe we're intentionally removing timezone information.  I believe we're intentionally prefer to rely on the built-in `ISO` conversion rather than crafting our own.  And yes, exactly how you put it - by using `toISOString()`, it takes any timezone-aware format and converts it to `UTC` (adds a `Z` at the end).

Reply:
Works for me 👍